packages/Datadog.Unity/Runtime/Android/DatadogAndroidPlatform.cs

```diff
@@ -52,12 +52,14 @@ public void Init(DatadogConfigurationOptions options)
                 environment = "prod";
             }
 
+            var serviceName = options.ServiceName == string.Empty ? null : options.ServiceName;
```

Comment:
Can this be null?

I wonder what happens if you send an event with no service name which it seems you were already.

https://github.com/DataDog/dd-sdk-ios/blob/b58fd2521194943969109e94801eb6154bb555ea/DatadogCore/Sources/Datadog.swift#L423C1-L424C1

We always had a value.

Reply:
Yes it can be null on Android (which is what it was before). Android will fill in the default if null is passed in.